### PR TITLE
Bug/gh929 behavior fixes

### PR DESCRIPTION
**Pull request by SimonWoodRhea**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/RHEAGROUP/COMET-IME-Community-Edition/pulls) open
- [x] I have verified that I am following the COMET-IME [code style guidelines](https://raw.githubusercontent.com/RHEAGROUP/COMET-IME-Community-Edition/master/.github/CONTRIBUTING.md)
- [ ] I have provided test coverage for my change (where applicable)

### Description
I have removed tokens {colon} in the header text. 

Fixes:
When adding a parameter to an existing behavior exception was thrown.
Tooltips for parameter mapping buttons
Null reference exception when validating parameter name field

### CODE REVIEW

**Review by alexatrhea — APPROVED** [review]

File: CDP4Composition/CommonView/Resources/ScriptTemplate.xaml [hunk 1]
```diff
@@ -31,11 +31,11 @@
                     <dxb:BarButtonItem x:Name="AddParameterButton"
                                        Command="{Binding AddParameterCommand}"
                                        Glyph="{dx:DXImage Image=Add_16x16.png}"
-                                       Hint="Create a File Type" />
+                                       Hint="Create a parameter mapping" />
```
Comment (lxatrhea) [inline CDP4Composition/CommonView/Resources/ScriptTemplate.xaml:34]: Please use Capital for Parameter: `Parameter mapping`

File: CDP4Composition/CommonView/Resources/ScriptTemplate.xaml [hunk 2]
```diff
@@ -31,11 +31,11 @@
...
                     <dxb:BarButtonItem x:Name="DeleteParameterButton"
                                        Command="{Binding DeleteParameterCommand}"
                                        Glyph="{dx:DXImage Image=Delete_16x16.png}"
-                                       Hint="Remove the selected File Type" />
+                                       Hint="Remove the selected parameter mapping" />
```
Comment (lxatrhea) [inline CDP4Composition/CommonView/Resources/ScriptTemplate.xaml:38]: Please use Capital for Parameter: `Parameter mapping`

File: CDP4Composition/CommonView/ViewModels/BehaviorDialogViewModel.cs [hunk 3]
```diff
@@ -152,7 +152,7 @@ private IBehavioralModelKindViewModel CreateBehavioralModelKindViewModel(Behavio
                 case BehavioralModelKind.Other:
                     {
                         var elementDefinition = this.Container as ElementDefinition;
-                        var vm = new ScriptKindViewModel(this.Thing.BehavioralParameter, this.Thing.Script, this.Session, this, elementDefinition?.Parameter);
+                        var vm = new ScriptKindViewModel(this.Thing.BehavioralParameter.Select( t => t.Clone(false)), this.Thing.Script, this.Session, this, elementDefinition?.Parameter);
```
Comment (lxatrhea) [inline CDP4Composition/CommonView/ViewModels/BehaviorDialogViewModel.cs:155]: This line is rather long. Please add line breaks for every parameter, which is what ReSharper would suggest.

**Review by lxatrhea — CHANGES_REQUESTED** [review]
Small cosmetic changes requested.

**Pull request merged**